.travis.yml

```diff
@@ -34,8 +34,10 @@ matrix:
   include:
     - compiler: gcc
       env: TASK="test-core"
-    - env: TASK="test-extended"
-    - env: TASK="doc"
+    - compiler: clang
```

Comment:
Setting the compiler to clang is causing the tests all tests to segfault when they run. I have a gut feeling it's an issue with our dependencies being compiled with g++ and probably a different version of the std c++ lib. So I would suggest to leave this one as well to gcc.